src/lib/render-lit-html.ts

```diff
@@ -384,8 +409,15 @@ const getTemplate = (result: TemplateResult) => {
   return t;
 };
 
+type RenderableCustomElement = HTMLElement & {
```

Comment:
an interface that extends HTMLElement will let you remove the `new()` declaration.